### PR TITLE
DOPS-2746 Add sonar, dojo

### PR DESCRIPTION
**Pull request by C4tWithShell**
1. [Set up sonar](https://sonar.soramitsu.co.jp/dashboard?id=sora%3Asora2-network&branch=feature%2FDOPS-2746%2Fsonar_dojo)
2. Set up dojo
3. Fixed problem with coverage, change to lcov format

**Merge after defectdojo update!**

### CODE REVIEW

**Comment by Alexey-N-Chernyshov** [comment]
Contracts are moved to https://github.com/sora-xor/sora2-federated-bridge-evm-contracts.git, see #780.
There is no need to run Slither in this repo.

File: Jenkinsfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@Library('jenkins-library') _
+@Library('jenkins-library@feature/DOPS-2746/rust_sonar_dojo_slither') _
```
Comment (wpi86) [inline Jenkinsfile:1]: Forgot?

**Review by wpi86 — CHANGES_REQUESTED** [review]

**Review by Alexey-N-Chernyshov — CHANGES_REQUESTED** [review]
Solidity was moved to https://github.com/sora-xor/sora2-federated-bridge-evm-contracts in #685

**Comment by C4tWithShell** [comment]
> Contracts were moved to https://github.com/sora-xor/sora2-federated-bridge-evm-contracts.git, see #780. There is no need to run Slither in this repo.

Removed

File: Jenkinsfile (hunk 1 above)
Comment (C4tWithShell) [reply Jenkinsfile:1]: Nope, I leave it because we did not update defectDojo to support slither reports. 
Now it is. Removed

**Review by BAStos525 — APPROVED** [review]

**Review by wpi86 — APPROVED** [review]

**Review by alexkg — APPROVED** [review]

**Review by Alexey-N-Chernyshov — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

**Review by Cre-eD — APPROVED** [review]

**Comment by C4tWithShell** [comment]
Created PR for wrong branch. Created a new PR

**Pull request closed without merging**